index.js

```diff
@@ -12,7 +12,7 @@ function liftErrorCode (code) {
     return 1011
   } else if (code === 1004 || code === 1005 || code === 1006) {
     // ws module forbid those error codes usage, lift to "application level" (4xxx)
-    return 4000 + (code % 1000)
```

Comment:
why was the `% 1000)` part unnecessary?

Reply:
The if clause already narrows the values down to the numbers 1004, 1005 and 1006. 
So the % 1000 would just return the numbers 4, 5 and 6.

So we can save us the modulo and directly add 3000 instead of 4000.

Reply:
smart!

Reply:
thx